src/Bridges/Nette/TracyExtension.php

```diff
@@ -19,6 +19,8 @@ class TracyExtension extends Nette\DI\CompilerExtension
 {
 	public $defaults = array(
 		'email' => NULL,
+		'logDirectory' => NULL,
+		'logSeverity' => 0,
```

Comment:
Maybe for Tracy, but (generally speaking) it allows you to generate optimal code which in some cases may reduces number of included files.